po/pt_BR.po

```diff
@@ -0,0 +1,453 @@
+msgstr "Nenhum dispositivo emparelhado."
+
+#: lib/logitech_receiver/status.py:99
+msgid "1 paired device."
+msgstr "1 dispositivo emparelhado."
+
+#: lib/logitech_receiver/status.py:100
+msgid " paired devices."
+msgstr " dispositivos emparelhados."
+
+#: lib/logitech_receiver/status.py:149 lib/logitech_receiver/status.py:151
+#: lib/logitech_receiver/status.py:203 lib/logitech_receiver/status.py:205
+#: lib/solaar/ui/window.py:143
+msgid "Battery"
+msgstr "Bateria"
+
+#: lib/logitech_receiver/status.py:162 lib/solaar/ui/window.py:150
+msgid "Lighting"
+msgstr "Iluminação"
+
+#: lib/logitech_receiver/status.py:162 lib/solaar/ui/window.py:663
+msgid "lux"
+msgstr "lux"
+
+#: lib/solaar/listener.py:95
+msgid "The receiver was unplugged."
+msgstr "O receptor foi desplugado."
+
+#: lib/solaar/ui/__init__.py:48
+msgid "Permissions error"
+msgstr "Erro de permissão"
+
+#: lib/solaar/ui/__init__.py:49
+#, python-format
+msgid "Found a Logitech Receiver (%s), but did not have permission to open it."
+msgstr "Foi encontrado um Receptor da Logitech (%s), mas sem permissão para abri-lo."
+
+#: lib/solaar/ui/__init__.py:51
+msgid ""
+"If you've just installed Solaar, try removing the receiver and plugging it "
+"back in."
+msgstr ""
+"Se você acabou de instalar o Solaar, tente retirar o receptor e plugá-lo "
+"novamente."
+
+#: lib/solaar/ui/__init__.py:53
+msgid "Unpairing failed"
+msgstr "Desemparelhamento falhou"
+
+#: lib/solaar/ui/__init__.py:54
+#, python-brace-format
+msgid "Failed to unpair %{device} from %{receiver}."
+msgstr "Falha ao desemparelhar %{device} do %{receiver}."
+
+#: lib/solaar/ui/__init__.py:56
+msgid "The receiver returned an error, with no further details."
+msgstr "O receptor retornou um erro, sem maiores detalhes."
+
+#: lib/solaar/ui/about.py:39
+msgid ""
+"Shows status of devices connected\n"
+"through wireless Logitech receivers."
+msgstr ""
+"Mostra o status dos dispositivos conectados\n"
+"através de receptores sem fio da Logitech."
+
+#: lib/solaar/ui/about.py:48
+msgid "GUI design"
+msgstr "Design da GUI"
+
+#: lib/solaar/ui/about.py:49
+msgid "Testing"
+msgstr "Teste"
+
+#: lib/solaar/ui/about.py:54
+msgid "Logitech documentation"
+msgstr "Documentação da Logitech"
+
+#: lib/solaar/ui/action.py:68 lib/solaar/ui/window.py:316
+msgid "About"
+msgstr "Sobre"
+
+#: lib/solaar/ui/action.py:95 lib/solaar/ui/action.py:98
+#: lib/solaar/ui/window.py:203
+msgid "Unpair"
+msgstr "Desemparelhar"
+
+#: lib/solaar/ui/config_panel.py:98
+msgid "Working"
+msgstr "Funcionando"
+
+#: lib/solaar/ui/config_panel.py:101
+msgid "Read/write operation failed."
+msgstr "Operação de leitura/escrita falhou."
+
+#: lib/solaar/ui/notify.py:120
+msgid "connected"
+msgstr "conectado"
+
+#: lib/solaar/ui/notify.py:122 lib/solaar/ui/tray.py:290
+#: lib/solaar/ui/tray.py:295 lib/solaar/ui/window.py:653
+msgid "offline"
+msgstr "offline"
+
+#: lib/solaar/ui/pair_window.py:133
+msgid "Pairing failed"
+msgstr "Pareamento falhou"
```

Comment:
Should be "Emparelhamento falhou" since "Pareamento" is a tech jargon not traditional Portuguese word